tests/NuGet.IndexingTests/VerifiedPackagesParsingTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+using NuGet.Indexing;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+
+namespace NuGet.IndexingTests
+{
+    public class VerifiedPackagesParsingTests
```

Comment:
No strong opinion here. Generally, I try to test the entry point for complete coverage.

I thought it'd just require a simple mock to have the loader return a reader... but if it's more complex, wouldn't worry about it.